clients/src/test/java/org/apache/kafka/clients/MetadataTest.java

```diff
@@ -845,6 +887,7 @@ protected boolean retainTopic(String topic, boolean isInternal, long nowMs) {
         assertEquals(cluster.topics(), new HashSet<>(Arrays.asList("oldValidTopic", "keepValidTopic")));
         assertEquals(cluster.partitionsForTopic("oldValidTopic").size(), 2);
         assertEquals(cluster.partitionsForTopic("keepValidTopic").size(), 3);
+        assertTrue(cluster.topicIds().containsAll(topicIds.values()));
```

Comment:
Do we want to make this assertion stronger? Or is `topicIds` a subset?

Reply:
To make it stronger, do we just also assert the inverse `(topicIds.values().containsAll(cluster.topicIds())` Or am I being silly and we can just check equality here?

Reply:
The collection types are different so I made a helper method to test both containAll. There may have been a helper somewhere else in the code but I couldn't find it.